kani-compiler/src/codegen_cprover_gotoc/codegen/function.rs

```diff
@@ -96,13 +95,14 @@ impl<'tcx> GotocCtx<'tcx> {
             debug!("Skipping function {}", self.current_fn().readable_name());
             self.codegen_function_prelude();
             self.codegen_declare_variables();
-            let body = self.codegen_fatal_error(
-                PropertyClass::UnsupportedConstruct,
-                &GotocCtx::unsupported_msg(
-                    &(String::from("The function ") + self.current_fn().readable_name()),
-                    None,
-                ),
-                Some(self.current_fn().mir().span),
+            let loc = self.codegen_span(&self.current_fn().mir().span);
+            let readable_name = format!("The function {}", self.current_fn().readable_name());
+            // We'll ideally just get rid of this eventually, but use "mimic" to avoid extra compilation warnings
+            let body = self.codegen_mimic_unimplemented(
+                &readable_name,
+                loc,
+                // There actually are links to specific issues in `should_skip_current_fn`...
```

Comment:
Should we keep a map from fn-name to url?

Reply:
Make `should_skip_current_fn` 

```rust
    fn should_skip_current_fn(&self) -> Option<&str> {
        let current_fn_name =
            with_no_trimmed_paths!(self.tcx.def_path_str(self.current_fn().instance().def_id()));
        // We cannot use self.current_fn().readable_name() because it gives the monomorphized type, which is more difficult to match on.
        // Ideally we should not rely on the pretty-printed name here. Tracked here: https://github.com/model-checking/kani/issues/1374
        match current_fn_name.as_str() {
            // https://github.com/model-checking/kani/issues/202
            "fmt::ArgumentV1::<'a>::as_usize" => Some("https://github.com/model-checking/kani/issues/202"),
           // ...
            _ => None,
        }
    }
```

Reply:
I thought about sneaking this change in here but there are some other un-linked skips in there, and it seemed like too much to figure out now.